Makefile

```
@@ -70,3 +70,11 @@ uninstall:
 .PHONY: clean
 clean:
 	rm -f bin/*
+
+dev/mediorum:
```

Since this is more of a 'command' and not an actual target file, let's make it phony and prefix it with `mediorum` so that it's easy to autocomplete

```suggestion

.PHONY: mediorum-dev
mediorum-dev:
```